identity/app/views/fragments/profile/membershipForm/upsell.scala.html

```diff
@@ -0,0 +1,8 @@
+<div class="membership-up-sell is-hidden js-mem-up-sell">
+    <div class="membership-up-sell__heading">
+        <h2 class="membership-header">Join Guardian Membership</h2>
+    </div>
+    <p class="membership-up-sell__content">We're bringing the Guardian to life through live events and meet-ups Join and be part of the conversations that matter</p>
```

Comment:
That must be a mistake then (copy/paste fail) – that's definitely two sentences. A full stop after "meet-ups" would work if it all has to be one line.